test/tools/utils.ts

```diff
@@ -87,8 +96,13 @@ export class EventCollector {
    * Will only return one event at a time from the front of the list
    * Useful for iterating over the events in the order they occurred
    */
-  waitAndShiftEvent(eventName: string): Promise<Record<string, any>> {
-    return new Promise<Record<string, any>>((resolve, reject) => {
+  waitAndShiftEvent(eventName) {
```

Comment:
I believe this is now no longer used with the Promise.race having been removed

Reply:
Removed!